### DISCUSSION TITLE
Parse JSON data in Python [May 29]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2270] [Discord Threads]
> 🙋‍♂️ **Tastier Sub**

**Comment by o1lab** [comment]
> 🙋‍♂️ **FrancyMak**  
#### Hi! So.   
####    
####    
#### ```py   
#### import json   
####    
#### person = '{"name": "Bob", "languages": ["English", "French"]}'   
#### person_dict = json.loads(person)   
####    
#### # Output: {'name': 'Bob', 'languages': ['English', 'French']}   
#### print( person_dict)   
####    
#### # Output: ['English', 'French']   
#### print(person_dict['languages'])   
#### ```  
#  
> 🙋‍♂️ **FrancyMak**  
#### If you want to use a file:   
####    
#### ```py   
#### import json   
####    
#### with open('path_to_file/person.json', 'r') as f:   
####   data = json.load(f)   
####    
#### # Output: {'name': 'Bob', 'languages': ['English', 'French']}   
#### print(data)   
#### ```  
#  
> 🙋‍♂️ **FrancyMak**  
#### Don’t forget the “r” after the file path, it means “read”  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/979798488978178049)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
